### PR TITLE
Add release notes for 0.260.1

### PR DESCRIPTION
**Pull request by yuanzhanhku**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.260.1.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.260.1
+===============
+
+General Changes
+_______________
+* Fix a concurrent issue which could cause query failures. (:pr:`16664`)
+* Disable the AGG IF to FILTER rewrite rule by default as it could cause query failures in some corner cases. (:pr:`16661`)
```
Comment (highker) [inline presto-docs/src/main/sphinx/release/release-0.260.1.rst:8]: If we could be more specific on "some" or "could" part, that will help with our customers to understand the impact. Would love to see @mayankgarg1990's comments on this.

Comment (yuanzhanhku) [reply presto-docs/src/main/sphinx/release/release-0.260.1.rst]: Added more details to the note to explain which cases could trigger the issues.

File: presto-docs/src/main/sphinx/release/release-0.260.rst [hunk 2]
```diff
@@ -2,6 +2,11 @@
 Release 0.260
 =============
 
+.. warning::
+
+   There is a concurrent issue in the metric tracking framework in this release which could cause query failures when Alluxio caching is enabled.
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.260.rst:7]: `concurrency issue`

File: presto-docs/src/main/sphinx/release/release-0.260.1.rst [hunk 3]
```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.260.1
+===============
+
+General Changes
+_______________
+* Fix a concurrent issue in the metric tracking framework which could cause query failures when alluxio caching is enabled. (:pr:`16664`)
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.260.1.rst:7]:
```
Fix query failures caused due to a concurrency issue in the runtime metrics framework when Alluxio caching is enabled. (:pr:`16664`)
```

File: presto-docs/src/main/sphinx/release/release-0.260.1.rst [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+* Disable the AGG IF to FILTER rewrite rule by default as it could cause query failures when the IF branch returns exceptions for rows not meeting the IF condition, e.g, SUM(IF(CARDINALITY(array) > 0, array[1])). (:pr:`16661`)
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.260.1.rst:8]:
```
* Fix query failures due to ``AGG_FUNCTION(IF())`` to ``FILTER`` rewrite rule by disabling the rule by default. (:pr:`16661`)
```

File: presto-docs/src/main/sphinx/release/release-0.260.rst [hunk 5]
```diff
@@ -2,6 +2,11 @@
...
+   optimizer.aggregation-if-to-filter-rewrite-enabled is enabled by default in this release which could cause query failures when the IF branch returns exceptions for rows not meeting the IF condition, e.g, SUM(IF(CARDINALITY(array) > 0, array[1])).
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.260.rst:8]: use double back quotes for expressions

**Review by mayankgarg1990 — CHANGES_REQUESTED** [review]

File: presto-docs/src/main/sphinx/release/release-0.260.rst (hunk 2 above)
Comment (yuanzhanhku) [reply presto-docs/src/main/sphinx/release/release-0.260.rst]: Done

File: presto-docs/src/main/sphinx/release/release-0.260.1.rst (hunk 3 above)
Comment (yuanzhanhku) [reply presto-docs/src/main/sphinx/release/release-0.260.1.rst]: Updated

File: presto-docs/src/main/sphinx/release/release-0.260.1.rst (hunk 4 above)
Comment (yuanzhanhku) [reply presto-docs/src/main/sphinx/release/release-0.260.1.rst]: Updated

File: presto-docs/src/main/sphinx/release/release-0.260.rst (hunk 5 above)
Comment (yuanzhanhku) [reply presto-docs/src/main/sphinx/release/release-0.260.rst]: Done.

File: presto-docs/src/main/sphinx/release/release-0.260.rst [hunk 6]
```diff
@@ -2,6 +2,11 @@
 Release 0.260
 =============
 
+.. warning::
+
+   There is a concurrency issue in the metric tracking framework in this release which could cause query failures when Alluxio caching is enabled.
+   optimizer.aggregation-if-to-filter-rewrite-enabled is enabled by default in this release which could cause query failures when the IF branch returns exceptions for rows not meeting the IF condition, e.g, ``SUM(IF(CARDINALITY(array) > 0, array[1]))``.
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.260.rst:8]: double back quote the property name as well.

**Review by mayankgarg1990 — CHANGES_REQUESTED** [review]

File: presto-docs/src/main/sphinx/release/release-0.260.rst (hunk 6 above)
Comment (yuanzhanhku) [reply presto-docs/src/main/sphinx/release/release-0.260.rst]: Done.

**Review by mayankgarg1990 — APPROVED** [review]

**Pull request merged**